.github/workflows/coordinated-merge.yaml

```
@@ -0,0 +1,114 @@
+name: Coordinated Merge for FoamAdapter + NeoN
+
+on:
+  pull_request:
+    types: [labeled, unlabeled, synchronize, reopened, opened]
+
+jobs:
+  coordinated-merge:
+    if: contains(github.event.pull_request.labels.*.name, 'coordinated-merge')
+    runs-on: ubuntu-latest
+    env:
+      GH_TOKEN: ${{ secrets.MERGE_TOKEN }}
+      NEON_REPO: exasim-project/NeoN
+      FOAM_REPO: exasim-project/FoamAdapter
```

Depending on whether we already merged #146 
```suggestion
      FOAM_REPO: exasim-project/NeoFOAM
```